### DISCUSSION TITLE
Optional argument

### DISCUSSION THREAD

**Discussion by onLaait** [Minestom/Minestom#1689] [Help]
![2023-01-14_05 40 31](https://user-images.githubusercontent.com/98655155/212420336-dcdab6ff-3052-41ca-99cd-673236618f84.png)

In vanilla, optional arguments are wrapped in '[' and ']' symbols, but how can i do this in minestom?

**Comment by GoldenStack** [comment]
You can use Argument#setDefaultValue.
Here's the method where you provide an object; you can also give it a supplier.
https://javadoc.minestom.net/net/minestom/server/command/builder/arguments/Argument.html#setDefaultValue(T)
